documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -16,12 +16,13 @@ Supported authentication types:
 * `scram-sha-512` for SCRAM-SHA-512 authentication
 
 If `tls` or `scram-sha-512` is specified, the User Operator creates authentication credentials when it creates the user.
-If `tls-external` is specified, the user is expected to use TLS client authentication, but no authentication credentials are created.
+If `tls-external` is specified, the user still uses TLS client authentication, but no authentication credentials are created.
+Use this option when you're providing your own certificates.
 When no authentication type is specified, the User Operator does not create the user or its credentials.
 
-Use `tls-external` to authenticate with TLS using a certificate issued outside the User Operator.
+You can use `tls-external` to authenticate with TLS using a certificate issued outside the User Operator.
```

Comment:
I think it could help to make it more clear that if you use this, issuing the certificate is users reposnsiiblity.

```suggestion
You can use `tls-external` to authenticate with TLS Client Authentication using a certificate issued on your own.
```